### PR TITLE
Add TLS support in scram-proxy

### PR DESCRIPTION
**Pull request by SergeyMelnikov**
This requires https://github.com/neondatabase/helm-charts/pull/12 to be released first.

### CODE REVIEW

**Review by kelvich — APPROVED** [review]

File: .circleci/helm-values/staging.proxy-scram.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ image:
 settings:
   authBackend: "console"
   authEndpoint: "http://console-staging.local:9095/management/api/v2"
```
Comment (cicdteam) [inline .circleci/helm-values/staging.proxy-scram.yaml:9]: by the way, this is wrong. AWS ALB do not serve 9095 port, it serve 80 (and pass traffic to port 9095 in target pods) only for internal console hostnames.

It should be simple `authEndpoint: "http://console-staging.local/management/api/v2"`

**Review by cicdteam — CHANGES_REQUESTED** [review]

**Review by cicdteam — APPROVED** [review]

**Comment by hlinnaka** [comment]
Cool!

I tried:

```
~$ psql postgres://hlinnaka%40zenith@start.stage.neon.tech/fragrant-rice-942513?sslmode=require
psql: error: connection to server at "start.stage.neon.tech" (35.169.157.111), port 5432 failed: server does not support SSL, but SSL was required
```

And:
```
~$ psql postgres://hlinnaka%40zenith@fragrant-rice-942513.cloud.stage.neon.tech/fragrant-rice-942513?sslmode=require
psql: error: connection to server at "fragrant-rice-942513.cloud.stage.neon.tech" (3.234.65.145), port 5432 failed: ERROR:  Internal error
```

There are more steps needed to have TLS working?

**Comment by SergeyMelnikov** [comment]
In proxy logs for your attempt
```
got message: SslRequest
got message: StartupMessage { major_version: 3, minor_version: 0, params: {"application_name": "psql", "": "", "user": "hlinnaka@zenith", "database": "fragrant-rice-942513", "client_encoding": "UTF8"} }
cplane request: http://console-staging.local/management/api/v2/proxy_get_role_secret?cluster=fragrant-rice-942513&role=hlinnaka%40zenith
error: Bad authentication secret
```
Looks like something is wrong with communication back to console, cc @kelvich.

**Comment by kelvich** [comment]
I've managed to connect to the cluster with scram+ssl. There is some bug about waking up compute in the new flow. You can try to create a new cluster, the click "generate _password_" and connect via `psql postgres://hlinnaka%40zenith:ASDFASDF@fragrant-rice-942513.cloud.stage.neon.tech/postgres`. Looking at this bug now.

**Pull request merged**